### ISSUE TITLE
Error when deploying iOS app (dev): Invariant Violation: unrecognized selector sent to instance "setWindow"

### ISSUE THREAD

**Issue by aleBiriri** [unimodules/react-native-unimodules#166]
I wanted to describe an error i encountered and its solution, in order to help anyone who can need it.

I was trying to run a React Native iOS application in my iPhone, but then XCode showed the following error message:

**Invariant Violation: unrecognized selector sent to instance "setWindow"**, followd by some debug traces.

In order to solve this, and following what is described in this [link](https://stackoverflow.com/questions/34960068/auto-property-synthesis-will-not-synthesise-property-description-because-it-is/34960300), i found a solution: Add the statement "**@synthesize window;**" in ios/MyApp/AppDelegate.m, just after the line "@implementation AppDelegate".

In a nutshell, the problem was that the window variable is declared as readonly in parent class of AppDelegate. Since, it is necessary to "synthesize it" to make it writable and set its new value when the app launchs.

Hope it can be useful for someone.

Tested in React Native: 0.62.2 and 0.64
XCode: 12.4
